### PR TITLE
Allow %f in pythonpath for linting relative imports

### PR DESCRIPTION
**Pull request by rainyday**
[Review on Reviewable](https://reviewable.io/reviews/atomlinter/linter-pylint/98)

### CODE REVIEW

**Comment by SpainTrain** [comment]
Clever.

Relative imports should work without special configuration, which is was the aim of https://github.com/AtomLinter/linter-pylint/pull/61.

  How about just adding `path.dirname(file)` to the `join` on line 81 so one doesn't have to change one's config to lint same dir relative imports?

**Comment by rainyday** [comment]
Yeah that makes more sense. I'll leave the option in though just in case someone has a weird configuration.

**Comment by SpainTrain** [comment]
:+1:

**Comment by rainyday** [comment]
This should be ready for merge

File: lib/main.coffee [hunk 1]
```diff
@@ -73,12 +74,13 @@ module.exports =
         return helpers.tempFile path.basename(file), activeEditor.getText(), (tmpFilename) =>
           # default project dir to file directory if path cannot be determined
           projDir = @getProjDir(file) or path.dirname(file)
-          cwd = @cwd.replace(/%f/g, path.dirname(file)).replace(/%p/g, projDir)
+          fileDir = path.dirname(file)
```
Comment (SpainTrain) [inline lib/main.coffee]: :+1:   I was going to ask for this, but didn't want to be nit picky :grin:

File: lib/main.coffee [hunk 2]
```diff
@@ -13,17 +13,18 @@ module.exports =
     pythonPath:
       type: 'string'
       default: ''
-      description: 'Paths to be added to $PYTHONPATH. Use %p for current project directory.'
+      description: 'Paths to be added to $PYTHONPATH. Use %p for current project directory or % for the directory of
```
Comment (SpainTrain) [inline lib/main.coffee]: tpyo: I think you want `%f`

**Comment by rainyday** [comment]
just pushed a fixup for a typo, should actually be done this time!

**Comment by SpainTrain** [comment]
I saw, you pushed it right as I was leaving the comment :laughing:

**Comment by SpainTrain** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/atomlinter/linter-pylint/98)*

**Comment by SpainTrain** [comment]
published thanks!

**Pull request merged**